### ISSUE TITLE
datadog_tracking_http_client required Dart version is too high

### ISSUE THREAD

**Issue by arcticfox1919** [DataDog/dd-sdk-flutter#155]
```
The current Dart SDK version is 2.15.1.
depends on datadog_tracking_http_client >=1.0.0-beta.1 which requires SDK version >=2.16.1 <3.0.0, version solving failed.
```

**Comment by fuzzybinary** [comment]
It looks like it should be safe to drop the version constraint down to 2.15.0 (which is what it is for the main plugin).

I'll look into fixing that and pushing a release.

**Comment by fuzzybinary** [comment]
datadog_tracking_http_client v1.0.0-beta.2 is now published and should support Dart 2.15.
https://pub.dev/packages/datadog_tracking_http_client/versions/1.0.0-beta.2

Note that 1.0.1-beta.1 still requires Dart 2.17 / Flutter 3, but the functionality of the releases is the same otherwise.
